lib/validators/helpers.js

```diff
@@ -36,7 +38,7 @@ api.createDidDocumentLoader = ({
       // FIXME: do a better check than searching for `:nym:`
       if(did.includes(':nym:')) {
         // generate nym DIDs, do not use record
-        didDocument = (await VeresOneDidDoc.fromNym({did})).doc;
+        ({didDocument} = await fromNym({did, cryptoLd, verificationSuite}));
```

Comment:
We're passing in `cryptoLd`? Not a fan of dependency injection here. But, not holding up the PR for it either.

Reply:
Since `fromNym` is a standalone function from a plain module (just as you prefer!), there's no other good method from passing in a configured (with selected 2020 or 2018 suites) `cryptoLd` instance, than this dependency injection.

Reply:
@dmitrizagidulin, I suspect that `cryptoLd` isn't even used in `fromNym`. I didn't understand what it would be used for if the verification suite is already present. I went looking and saw that it is passed here (along with `invokeKey`):

https://github.com/veres-one/did-veres-one/blob/7b798da9b3cbb0d0e0957a6d2c25719aa177eaeb/lib/VeresOneDriver.js#L650

Which is then not used here (due to `invokeKey` needing to exist -- as the key itself is in the `nym`, which is the point of cryptonyms):

https://github.com/veres-one/did-veres-one/blob/7b798da9b3cbb0d0e0957a6d2c25719aa177eaeb/lib/VeresOneDriver.js#L200-L201

So, we should try not passing `cryptoLd` here at all and see if everything continues to work just fine.

Reply:
If it does work, a PR should be made to did-veres-one to remove the `cryptoLd` parameter from `fromNym` -- as it shouldn't be needed.

Reply:
Also I notice that the `verificationSuite` is coming from the driver already, so `fromNym` could just be called on the driver and without passing anything but the `did` to resolve.

Reply:
+1, agreed.